### PR TITLE
GA notification

### LINKED ISSUE #1778: Update the rc breaking changes prompt for 2.0 GA

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1778 

DO NOT MERGE until after the `separate-cloud` branch has been merged and 2.0 has been announced.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: data/notifications.yaml [hunk 1]
```diff
@@ -7,14 +7,17 @@
...
-- id: rc-upgrade
-  level: warn
+- id: v2-ga
+  level: note
   scope:
-    - /influxdb/v2.0/
+    - /
   message: |
-    ### InfluxDB OSS 2.0 release candidate
+    ### InfluxDB OSS 2.0 now generally available
```
Comment (pierwill) [inline data/notifications.yaml:15]: I think an exclamation point would be appropriate. 😇

File: data/notifications.yaml [hunk 2]
```diff
@@ -7,14 +7,17 @@
...
 
-    **InfluxDB OSS v2.0.rc** includes **breaking changes** that require a manual
-    upgrade from all alpha and beta versions. For information, see:
+    **InfluxDB OSS 2.0** is now generally available and ready for production use.
+    See the [InfluxDB OSS 2.0 release notes](/influxdb/v2.0/reference/release-notes/influxdb/)
```
Comment (pierwill) [inline data/notifications.yaml:18]: Period needed at the end of line 18.

**Review by pierwill — APPROVED** [review]

File: data/notifications.yaml (hunk 1 above)
Comment (pierwill) [reply data/notifications.yaml:15]: But I'm profligate with exclamation marks.

Comment (sanderson) [reply data/notifications.yaml]: Agreed

File: data/notifications.yaml (hunk 2 above)
Comment (sanderson) [reply data/notifications.yaml]: 🙏

**Review by kelseiv — APPROVED** [review]

**Pull request closed without merging**